gateway-export-plugin/src/main/java/com/ca/apim/gateway/cagatewayexport/tasks/explode/linker/JmsDestinationLinker.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2018. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+package com.ca.apim.gateway.cagatewayexport.tasks.explode.linker;
+
+import com.ca.apim.gateway.cagatewayconfig.beans.Bundle;
+import com.ca.apim.gateway.cagatewayconfig.beans.JmsDestination;
+
+import javax.inject.Singleton;
+
+@Singleton
+public class JmsDestinationLinker implements EntityLinker<JmsDestination> {
+    
+    @Override
+    public Class<JmsDestination> getEntityClass() {
+        return JmsDestination.class;
+    }
+
+    @Override
+    public void link(JmsDestination entity, Bundle bundle, Bundle targetBundle) {
+        // TODO
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=138&issues=AWc81Lh6fimTzgTvo62A&open=AWc81Lh6fimTzgTvo62A)

[ISSUE_KEY]: AWc81Lh6fimTzgTvo62A